### PR TITLE
Docker: Improve Prometheus integration

### PR DESCRIPTION
**Pull request by gsiv**

### CODE REVIEW

File: prometheus/prometheus.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+global:
+  scrape_interval: 5s
+  scrape_timeout: 5s
```
Comment (ostcar) [inline prometheus/prometheus.yml:3]: At least for production, 1 minute should be enough.

File: caddy/Caddyfile [hunk 2]
```diff
@@ -13,5 +13,7 @@ import endpoint
 
     reverse_proxy /media/* media:8000
 
+    reverse_proxy /prometheus/* prometheus:9090
```
Comment (ostcar) [inline caddy/Caddyfile:16]: Is there some sort of restriction? I don't think, this URL should be public.

File: prometheus/prometheus.yml (hunk 1 above)
Comment (gsiv) [reply prometheus/prometheus.yml:3]: Thanks, I see.  So we should include two different configuration files.  I'll add one.

You presumably mean that `scrape_interval` should be set to 1 minute, correct?  Or should it be set for `scrape_timeout` as well?

File: caddy/Caddyfile (hunk 2 above)
Comment (gsiv) [reply caddy/Caddyfile:16]: As part of this setup, no.  My idea is to manage access using HTTP basic auth with the reverse proxy.

File: prometheus/prometheus.yml (hunk 1 above)
Comment (ostcar) [reply prometheus/prometheus.yml:3]: I would suggest to remove the values, so it uses the defaults.

scrape_interval 1m
scrape_timeout 10s

File: caddy/Caddyfile (hunk 2 above)
Comment (ostcar) [reply caddy/Caddyfile:16]: Ok

**Comment by gsiv** [comment]
I pushed changes with which I believe this patch is ready to be merged:

  - squashed commits
  - added warning to `.env`
  - removed the scrape settings for the production config

**Review by ostcar — APPROVED** [review]
Did not test it

**Review by emanuelschuetze — APPROVED** [review]

**Pull request merged**